discovery-provider/plugins/network-monitoring/Dockerfile.test

```diff
@@ -0,0 +1,13 @@
+FROM node:16
```

Comment:
whats purpose of this?

Reply:
It makes tests much easier - it's basically just the network monitoring job without the CRON job.

Reply:
tests?

Reply:
Not formal tests but just testing to make sure everything works locally. It's just a Dockerfile that runs a single network monitoring job because the main Dockerfile that we'll use in prod wraps everything in a CRON job

Reply:
ah got it, ok - can you leave comment indicating that